### ISSUE TITLE
Wiki and README

### ISSUE THREAD

**Issue by mautamu** [leftwm/leftwm#218]
Good afternoon,

I have been working on updating the Wiki with some new information: https://github.com/mautamu/leftwm_wiki . Would y'all please look it over and see if the changes are beneficial?

I am also working on updating the README to include leftwm-theme and some more recent changes:
https://github.com/mautamu/leftwm/tree/README

Any feedback would be much appreciated prior to PRing.

Best,
Mautamu

**Comment by AethanFoot** [comment]
The wiki entries look good, most definitely need a more comprehensive wiki (especially with theme configuration).

I would say that the README needs reordering, I would say it should go like this:
 Why go left (intro)
Dependencies (need them before I install. Maybe add extra popular package managers, e.g. pacman, dnf, etc)
Install/setup (as I dont need to know any config until I have installed it)
Themes (core element that should be setup first after install)
config stuff  (can only tinker once it is setup)

**Comment by VuiMuich** [comment]
You are doing awesome work mautamu!
And at this point it is also worth mentioning that I absolutely love the expample keybind!

I think it might be worth to mention that at least a `modifier = []` is needed with a keybind.
In the solution for keybind not working your suggest to open a PR, didn't you mean "open an issue"?

+1 on AethanFoot's suggestion

**Comment by VuiMuich** [comment]
On the `modifier = []`: what I actually meant was that you need this line specifically, even if you don't use a modkey in this keybind.
Like this won't work:
```toml
[[keybind]]
command = "Execute"
value = "pactl set-sink-mute $(pactl list short sinks|awk '{print $1}') toggle"
key = "XF86XK_AudioMute"
```

**Comment by mautamu** [comment]
Good catch on the PR/Issue switchup. I agree with AethanFoot's layout as well, and have rearranged the page to match. 

I have put a note in both the header and the troubleshooting sections to the effect of "you must include a modifier, even if `modifier=[]`", should we include it elsewhere as well? 

The discussion page may be helpful.
